improver_tests/ensemble_copula_coupling/test_utilities.py

```diff
@@ -398,5 +402,79 @@ def test_multiple_timesteps(self):
         self.assertArrayAlmostEqual(reshaped_array, expected)
 
 
+numba_installed = True
+try:
+    importlib.util.find_spec("numba")
+    from improver.ensemble_copula_coupling.numba_utilities import fast_interp_same_y
+except ImportError:
+    numba_installed = False
+
+
+class TestInterpolateMultipleRows(IrisTest):
```

Comment:
Perhaps this would be better named `Test_interpolate_multiple_rows_same_y`?

Reply:
It might be worth adding a test which mocks both `fast_interp_same_y` and `slow_interp_same_y` to check which implementation is actually used when numba is available and which one when not (i.e. when called via `interpolate_multiple_rows_same_y`).  Happy to provide some pointers if required.

Reply:
I have made an attempt at this, but I'm not very familiar with mocking, so suggestions for improvement are welcome.